deploy/StableJoeStaking.js

```diff
@@ -32,7 +39,13 @@ module.exports = async function ({ getNamedAccounts, deployments }) {
         execute: {
           init: {
             methodName: "initialize",
-            args: [rewardToken, joeAddress, feeCollector, depositFeePercent],
+            args: [
+              rewardToken,
+              joeAddress,
+              feeCollector,
+              depositFeePercent,
+              smolJoes,
+            ],
           },
         },
       },
```

Comment:
For safety reasons, don't forget to initialize the implementation even if for transparent proxy this is useless.